docker-compose/docker-compose.yaml

```diff
@@ -53,4 +53,31 @@ services:
     - /:/rootfs:ro
     - /var/run:/var/run:rw
     - /sys:/sys:ro
-    - /var/lib/docker/:/var/lib/docker:ro
\ No newline at end of file
+    - /var/lib/docker/:/var/lib/docker:ro
+
+  #   mysqld:
```

Comment:
We do not need `mysqld`. This is to run a DB. In reality, the customer will already have a DB running somewhere and they will just need info to start monitoring it.